### PR TITLE
Add (DocumentDB) to SQL description

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Agreed to in #520 but got left out.

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
image

File: src/experiences.ts [hunk 1]
```diff
@@ -48,10 +48,10 @@ export function getExperienceQuickPick(api: API): IAzureQuickPickItem<Experience
 }
 
 const experiencesArray: Experience[] = [
-    { api: API.DocumentDB, longName: "SQL", shortName: "SQL", kind: DBAccountKind.GlobalDocumentDB },
+    { api: API.DocumentDB, longName: "SQL", description: "(DocumentDB)", shortName: "SQL", kind: DBAccountKind.GlobalDocumentDB },
```
Comment (PrashanthCorp) [inline src/experiences.ts]: 'SQL (DocumentDB)' in one place and 'DocumentDB (SQL)' everywhere else seems a bit confusing to the user. 
For the sake of consistency, I think the order here should be swapped, considering what's done in the other PR.

Comment (EricJizbaMSFT) [reply src/experiences.ts]: I agree with @PrashanthCorp that we should be consistent, but I prefer 'SQL (DocumentDB)' so this PR looks fine to me

Comment (StephenWeatherford) [reply src/experiences.ts]: I'll change the other PR.

**Pull request merged**